VERSION

```diff
@@ -1 +1 @@
-0.10.4-dev-1
\ No newline at end of file
+0.10.4-dev-2
```

Comment:
I think that we need to strip the `\n`

Reply:
but not sure if that issue was fixed

Reply:
we should keep the newline

Reply:
we should keep the newline

Reply:
In the past it had compilation errors when we added it, if it works now with the newline I am ok with it